CMakeLists.txt

```diff
@@ -101,7 +99,6 @@ option(VELOX_ENABLE_CCACHE "Use ccache if installed." ON)
 
 option(VELOX_BUILD_TEST_UTILS "Builds Velox test utilities" OFF)
 option(VELOX_BUILD_PYTHON_PACKAGE "Builds Velox Python bindings" OFF)
-option(VELOX_BUILD_BENCHMARKS "Builds Velox benchmarks" OFF)
```

Comment:
Hi @majetideepak, I note this build option is used here. Do we still need to remove it?
https://github.com/facebookincubator/velox/blob/main/Makefile#L106

Reply:
@PHILO-HE thanks for this pointer. I will fix this. The Makefile target says basic benchmarks, but seems to be build all. This needs to be fixed as well.